core/p2p/p2pv1/server.go

```diff
@@ -108,8 +114,26 @@ func (p *P2PServerV1) Init(cfg config.P2PConfig, lg log.Logger, extra map[string
 			p.staticNodes["xuper"] = peerids
 		}
 	}
-
+	p.connectToBootNodes()
```

Comment:
user could set both static nodes and bootstrap nodes?

Reply:
User doesn't need to set both `staticNodes` and `bootstrapNodes`.